emcc.py

```diff
@@ -1576,10 +1576,11 @@ def check(input_file):
       # memalign is used to ensure allocated thread stacks are aligned.
       shared.Settings.EXPORTED_FUNCTIONS += ['_memalign', '_malloc']
 
-      # dynCall_ii is used to call pthread entry points in worker.js (as
-      # metadce does not consider worker.js, which is external, we must
-      # consider it a user export, i.e., one which can never be removed).
-      building.user_requested_exports += ['dynCall_ii']
+      if not shared.Settings.WASM:
```

Comment:
`WASM` should always be true I think, so is this just commenting out the code?

(`-s WASM=0` is converted into `WASM=1, WASMJS=1` internally. It is a little confusing...)